bibliopixel/drivers/visualizer.py

```diff
@@ -1,11 +1,6 @@
-import time
-
-import os
-import platform
-import subprocess
-from network import DriverNetwork, socket
-import math
+import time, os, platform, subprocess, math
```

Comment:
Oh, that's so spammy!  :-)  But sure, why not?

Google used to do that.  On some files, the first page would just be imports.  I never quite saw the idea of "one per line" but w/e.